### ISSUE TITLE
v. 3.11.0 ignores -q flag

### ISSUE THREAD

**Issue by belorenz** [tcort/markdown-link-check#248]
```shell
lore_be@ts-070240l:~/bob_ws$ sudo markdown-link-check --version
3.11.0
lore_be@ts-070240l:~/bob_ws$ sudo markdown-link-check -q src/adas/driver/novatel/README.md

FILE: src/adas/driver/novatel/README.md
  [✓] https://www.boost.org
  [✓] https://github.com/wjwwood/serial
  [✓] https://cmake.org
  [✖] https://www.ros.org/wiki/catkin
  [✓] https://www.ros.org
  [✓] https://ros.org/doc/api/sensor_msgs/html/msg/NavSatFix.html
  [✓] https://ros.org/doc/api/nav_msgs/html/msg/Odometry.html
  [✓] http://docs.ros.org/api/sensor_msgs/html/msg/Imu.html
  [✓] https://hexagondownloads.blob.core.windows.net/public/Novatel/assets/Documents/Manuals/om-20000104/om-20000104.pdf
  [✓] https://github.com/GAVLab/novatel/tree/master/include/novatel
  [✓] mailto:david.bla@bla.com
  [✓] mailto:franz.bla@bla.de

  12 links checked.

  ERROR: 1 dead links found!
  [✖] https://www.ros.org/wiki/catkin → Status: 0

```

**Comment by pgrawehr** [comment]
This ticket is basically a duplicate of #246

**Comment by tcort** [comment]
should be fixed in 3.11.1. If not, please re-open.
